src/app/utils/isValidPortNumber.test.ts

```diff
@@ -0,0 +1,15 @@
+import { isValidPortNumber } from ".";
+
+it("returns true for any number between 0 and 65535", () => {
+  for (let i = 0; i <= 65535; i++) {
```

Comment:
That is a computationally expensive test for the function it tests.... Edge cases and some common ports might be enough. But I'm happy to merge if it runs quickly.

Reply:
Locally it seems to run quickly enough, but thought I'd get your thoughts first.